doc/source/whatsnew/v0.18.1.txt

```diff
@@ -164,3 +164,4 @@ Bug Fixes
 
 
 - Bug in ``pivot_table`` when ``margins=True`` and ``dropna=True`` where nulls still contributed to margin count (:issue:`12577`)
+- Bug in ``DataFrameGroupBy.rank`` yielding empty dataframe instead of raising error (:issue:`11759`)
```

Comment:
make it more user-friendly, e.g. ``bug in ``.groupby(....).rank(..)`` when? (``numeric_only=False``) right?